echoshader/new_version/echogram.py

```diff
@@ -210,10 +213,39 @@ def echogram_single_frequency(
 
         self.gram_opts["Image"]["clim"] = self.Sv_range_slider.value
 
-        plot = (
+        gram_plot = (
```

Comment:
I think it will be good to change all the `gram` to `echogram` in the code to avoid confusion.
```suggestion
        echogram_plot = (
```